spec/features/entity_page_spec.rb

```diff
@@ -168,6 +168,32 @@ def should_visit_entity_page(url)
     end
   end
 
+  describe "Relationship Tab - with category Membership" do
```

Comment:
you could probably delete the pending `describe "relationships tab" on line 197?